### PR TITLE
adds the updated Bosch logo

### LINKED ISSUE #588: Add Bosch sponsor logo to page
Hi @andrea-kyurchiev 

Please can you add this logo to our sponsor webpage:  https://innersourcecommons.org/about/sponsors/

1. The Bosch logo should be put in the 'Supporters' part of the webpage.

2. The Bosch logo should be placed in alphabetical order (i.e. after Adeo).

3. Bosch have asked that we do not:  'reduce the horizontal and vertical margins'.  Adding more is not a problem.
Bosch_symbol_logo_black_red

### PR DESCRIPTION
**Pull request by andrea-kyurchiev**
Closes #588

### CODE REVIEW

**Comment by CiaraFlanagan** [comment]
Thanks @andrea-kyurchiev 

I will merge once I get sign off today or Monday.

**Review by gruetter — APPROVED** [review]

**Review by CiaraFlanagan — APPROVED** [review]

**Pull request merged**
